scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,60 @@
+## Converting Workspace Dataset Attributes to Snapshot Data
```

Comment:
The target here isn't snapshot data but is catalog entry data. The distinction is that the current schema that the catalog UI accepts is not from TDR or snapshots but is a custom schema created for the data catalog. So this is an effort to map workspace properties to the catalog's schema.

A separate / associated effort would be to formally document what the data catalog's schema actually is. The schema is based on TIM (specifically [TerraDCAT_ap:Dataset](https://github.com/DataBiosphere/terra-interoperability-model/blob/master/releases/1.x/terra-core/TerraDCAT-AP.ttl#L167)

Reply:
Gotcha, updating this title :)